packages/browser/src/index-cjs.ts

```diff
@@ -0,0 +1,13 @@
+import Client from '@bugsnag/core/client'
+// @ts-ignore
+import Event from '@bugsnag/core/event'
+// @ts-ignore
+import Session from '@bugsnag/core/session'
+// @ts-ignore
+import Breadcrumb from '@bugsnag/core/breadcrumb'
```

Comment:
Assuming these ignores will be removed once we've finished conversion of the core package